### PR TITLE
fix(platform): remove unused module imports in ApprovalFlow component

### PR DESCRIPTION
**Pull request by valorkin**
## Description

It seems ComboboxModule, ToolbarModule was imported but never used in Approval flow

### CODE REVIEW

File: libs/platform/src/lib/approval-flow/tsconfig.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   },
   "angularCompilerOptions": {
     "strictInjectionParameters": false,
-    "strictTemplates": false
+    "strictTemplates": false,
+    "fullTemplateTypeCheck": false
```
Comment (mikerodonnell89) [inline libs/platform/src/lib/approval-flow/tsconfig.json:25]: Why this change?

File: libs/platform/src/lib/approval-flow/approval-flow.module.ts [hunk 2]
```diff
@@ -80,13 +79,15 @@ import { ApprovalFlowToolbarActionsComponent } from './approval-flow-toolbar-act
...
         CheckboxModule,
         MenuModule,
-        ComboboxModule,
         SelectModule,
         DatePickerModule,
         FormModule,
         DragDropModule,
         RadioModule,
         IllustratedMessageModule
+        // todo: unused
```
Comment (mikerodonnell89) [inline libs/platform/src/lib/approval-flow/approval-flow.module.ts:88]: I think we can safely remove these lines entirely

File: libs/platform/src/lib/approval-flow/tsconfig.json (hunk 1 above)
Comment (valorkin) [reply libs/platform/src/lib/approval-flow/tsconfig.json:25]: all this should be true, and it's in a roadmap
added this row so folks didn't forget to enable it later

**Comment by valorkin** [comment]
@mikerodonnell89 answered\removed

**Review by mikerodonnell89 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
🚀
